src/main/webapp/WEB-INF/tags/fileUploader.tag

```diff
@@ -0,0 +1,40 @@
+<%@ tag language="java" pageEncoding="UTF-8"%>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@attribute name="attachmentsTarget" type="org.mamute.model.Post" required="false"%>
+
+<c:if test="${env.get('feature.inhouse.upload')}">
+    <div class="uploader">
+        <label>${t['question.attachments']} - <a class="add-file" href="#">${t['question.attachments.newfile']}</a></label>
```

Comment:
MDN says nothing about it: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label

But I can change it if that's a problem. Should I use span?